index.bs

```diff
@@ -62,6 +66,7 @@ spec: WebCryptoAPI; urlPrefix: https://www.w3.org/TR/WebCryptoAPI/
 </pre> <!-- class=anchors -->
 
 <pre class="link-defaults">
+spec:infra; type:dfn; text:list
```

Comment:
@jyasskin 
> Would y'all file an issue in https://github.com/tabatkins/bikeshed/issues about generating a dependencies section earlier in the document?

done: https://github.com/tabatkins/bikeshed/issues/937

> I'm going to take this thread as an indication not to add "infra" to #dependencies unless y'all say otherwise.

ok, tho perhaps Dependencies section ought to have link to `#index-defined-elsewhere` ?